srcpkgs/oblibs/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'oblibs'
+pkgname=oblibs
+version=0.0.1.1
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=configure
+configure_args=" --prefix=/usr"
```

Comment:
no need for the space between " and -

Reply:
Will do. Should probably add a -doc package, 66 has html docs like s6-rc.

Reply:
please do

Reply:
Done, thanx!